Tests/OpenCombineTests/PublisherTests/ThrottleTests.swift

```diff
@@ -0,0 +1,848 @@
+//
+//  ThrottleTests.swift
+//  
+//
+//  Created by Stuart Austin on 14/11/2020.
+//
+
+#if !WASI
```

Comment:
Is there anything that prevents these tests from execution on WASI? As far as I undestand, `VirtualTimeScheduler` is available on this platform, or is there anything else that's unavailable?